third_party/maya/lib/usdMaya/translatorXformable.cpp

```diff
@@ -288,7 +288,19 @@ static bool _pushUSDXformOpToMayaXform(
                                 PxrUsdMayaXformStack::RotateOrderFromOpType<MEulerRotation::RotationOrder>(
                                         xformop.GetOpType());
                         MEulerRotation eulerRot(xValue[i], yValue[i], zValue[i], MrotOrder);
+
+                        // For all rotations except the first one, we can check for Euler flips.
+                        // The first rotation can't be validated against a previous value so we always take it as it is.
+                        if (args.GetUseEulerFilter() && i > 0)
+                        {
+                            size_t previousRotationIndex = i - 1;
+                            MEulerRotation previousRotation(xValue[previousRotationIndex],
+                                                            yValue[previousRotationIndex],
+                                                            zValue[previousRotationIndex], MrotOrder);
+                            eulerRot.setToClosestSolution(previousRotation);
+                        }
```

Comment:
These changes won't actually do nothing, because rotateAxis is internally stored as a quaternion, so 360 flips change nothing.  ie, try this in a python prompt:

```
cmds.setAttr('persp.rotateAxis', 720, 0, 0)
print cmds.getAttr('persp.rotateAxis')
```